ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -1555,6 +1598,8 @@ void handleBody(InputStream payload, MessageMetadata messageMetadata, MessageInf
           }
         }
         blobType = blobData.getBlobType();
+        isChunkCompressed = blobData.isCompressed();
+        logger.trace("In FirstGetChunk.handleBody(), is Chunk compressed = " + isChunkCompressed);
```

Comment:
nit: probably it can include some blobId info, otherwise it's hard to trace the log.

Reply:
Agreed.